builder-support/post-build

```diff
@@ -9,6 +9,14 @@ DESTDIR="built_pkgs/"
 
 mkdir -p ${DESTDIR}
 
+# We need the GNU version of tar for --transform
+[ -z "$tar" ] && tar=`which gtar tar | grep '^/' | head -1`
```

Comment:
You might want to redirect stderr to /dev/null because gtar doesn't exist on every Linux install and shows an ugly error.

Reply:
```
$ which gtar tar
/bin/tar
$
```

I see no ugly error, or am I missing something?

Reply:
On my Gentoo with `sys-apps/which` version 2.21:

```
$ which gtar tar
which: no gtar in (/home/ekohl/bin:/usr/x86_64-pc-linux-gnu/gcc-bin/7.3.0:/usr/lib/llvm/6/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/games/bin)
/bin/tar
```

Reply:
Indeed, on CentOS 7:
```
[root@89a0de709c3a /]# which xtar gtar > /dev/null
which: no xtar in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
[root@89a0de709c3a /]# which xtar gtar
which: no xtar in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
/usr/bin/gtar
[root@89a0de709c3a /]# which xtar gtar 2> /dev/null
/usr/bin/gtar
```